lib/web/databases.go

```diff
@@ -141,16 +117,37 @@ func (h *Handler) handleDatabaseCreate(w http.ResponseWriter, r *http.Request, p
 
 // updateDatabaseRequest contains some updatable fields of a database resource.
 type updateDatabaseRequest struct {
-	CACert string `json:"ca_cert,omitempty"`
+	CACert *string    `json:"caCert,omitempty"`
+	Labels []ui.Label `json:"labels,omitempty"`
+	URI    string     `json:"uri,omitempty"`
+	AWSRDS *awsRDS    `json:"awsRds,omitempty"`
 }
 
 func (r *updateDatabaseRequest) checkAndSetDefaults() error {
-	if r.CACert == "" {
-		return trace.BadParameter("missing CA certificate data")
+	if r.CACert != nil {
+		if *r.CACert == "" {
+			return trace.BadParameter("missing CA certificate data")
+		}
+
+		if _, err := tlsutils.ParseCertificatePEM([]byte(*r.CACert)); err != nil {
+			return trace.BadParameter("could not parse provided CA as X.509 PEM certificate")
+		}
+		return nil
```

Comment:
Should we return `nil` here? It would skip the checks below if the `CACert` stuff is correct

Reply:
gah i missed that thanks: https://github.com/gravitational/teleport/pull/21062/commits/73666f008542c3dce0415cf4ce2767b99871d3a7